src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/Tensor.cs

```diff
@@ -243,7 +240,7 @@ public static ref readonly TensorSpan<T> ConcatenateOnDimension<T>(int dimension
             }
             Span<T> dstSpan = MemoryMarshal.CreateSpan(ref destination._reference, (int)totalLength);
 
-            if (dimension == 0 || dimension == -1)
+            if (dimension is 0 or (-1))
```

Comment:
are these parens around the -1 required?

Reply:
Believe so, the fixer automatically applied them.